README.md

```diff
@@ -42,33 +60,72 @@ model = resnet50()
+# Now use compressed_model as a usual torch.nn.Module 
+# to fine-tune compression parameters along with the model weights
 
 # ... the rest of the usual PyTorch-powered training pipeline
 
 # Export to ONNX or .pth when done fine-tuning
-comp_ctrl.export_model("compressed_model.onnx")
+compression_ctrl.export_model("compressed_model.onnx")
 torch.save(compressed_model.state_dict(), "compressed_model.pth")
 ```
 
-For a more detailed description of NNCF usage in your training code, see [Usage.md](./docs/Usage.md). For in-depth examples of NNCF integration, browse the [sample scripts](#model-compression-samples) code, or the [example patches](#third-party-repository-integration) to third-party repositories.
+### Usage example with TensorFlow
+```python
+import tensorflow as tf
 
-For more details about the framework architecture, refer to the [NNCFArchitecture.md](./docs/NNCFArchitecture.md).
+from nncf import NNCFConfig
+from nncf.tensorflow import create_compressed_model, register_default_init_args
+
+# Instantiate your uncompressed model
+from tensorflow.keras.applications import ResNet50
+model = ResNet50()
+
+# Load a configuration file to specify compression
+nncf_config = NNCFConfig.from_json("resnet50_int8.json")
+
+# Provide dataset for compression algorithm initialization
+representative_dataset = tf.data.Dataset.list_files("/path/*.jpeg")
+nncf_config = register_default_init_args(nncf_config, representative_dataset, batch_size=1)
+
+# Apply the specified compression algorithms to the model
+compression_ctrl, compressed_model = create_compressed_model(model, nncf_config)
+
+# Now use compressed_model as a usual Keras model
+# to fine-tune compression parameters along with the model weights
+
+# ... the rest of the usual TensorFlow-powered training pipeline
+
+# Export to Frozen Graph, TensorFlow SavedModel or .h5  when done fine-tuning 
+compression_ctrl.export_model("compressed_model.pb", save_format='frozen_graph')
+```
+
+For a more detailed description of NNCF usage in your training code, see [this tutorial](docs/Usage.md). 
+For in-depth examples of NNCF integration, browse the [sample scripts](#model-compression-samples) code, or the [example patches](#third-party-repository-integration) to third-party repositories.
 
 
 ## Model Compression Samples
 
 For a quicker start with NNCF-powered compression, you can also try the sample scripts, each of which provides a basic training pipeline for classification, semantic segmentation and object detection neural network training correspondingly.
 
 To run the samples please refer to the corresponding tutorials:
-- [Image Classification sample](examples/torch/classification/README.md)
-- [Object Detection sample](examples/torch/object_detection/README.md)
-- [Semantic Segmentation sample](examples/torch/semantic_segmentation/README.md)
+
+- PyTorch samples:
+  - [Image Classification sample](examples/torch/classification/README.md)
```

Comment:
Link to OTE and short description are required!

Reply:
I have added, please take a look.